src/domain/_common/components/VariantButton.tsx

```diff
@@ -2,8 +2,8 @@ import React, { type ReactNode } from 'react';
 
 interface ButtonProps {
   isDisabled: boolean;
```

Comment:
```suggestion
  disabled?: boolean;
```
native button처럼 disabled 로 쓰는건 어떨까여..?
default값은 false로 줘서 매번 안줘도 되게끔...

Reply:
좋은 것 같습니다!